### PR TITLE
add document about how to deploy locally

### PR DESCRIPTION
**Pull request by xiaoxstz**
Summary of the changes (in less than 80 characters):

* Add steps about how to deploy locally to the readme file

Addresses https://github.com/NuGet/NuGetGallery/issues/9326

### CODE REVIEW

**Comment by dnfgituser** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=9333) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=9333) before we can accept your contribution.<br/><br/>:x: xiaoxstz [sign now](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=9333)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/NuGet/NuGetGallery?pullRequest=9333) it.</sub>

File: README.md [hunk 1]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
 Refer to [our documentation](./docs/) for information on how to develop the frontend, use AAD, and more.
 
 ## Deploy
+### Deploy to Azure
 
 You will find instructions on how to deploy the Gallery to Azure [here](https://github.com/NuGet/NuGetGallery/blob/master/docs/Deploying/README.md).
 
+### Deploy locally
+After you succeed in running the NuGet Gallery, you can create a publish to deploy locally(such as your windows computer).
```
Comment (joelverhagen) [inline README.md:35]:
```suggestion
After you succeed in running the NuGet Gallery, you can create a Publish configuration to deploy locally (such as your local Windows computer). 
```

File: README.md [hunk 2]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
...
+### Deploy to Azure
 
 You will find instructions on how to deploy the Gallery to Azure [here](https://github.com/NuGet/NuGetGallery/blob/master/docs/Deploying/README.md).
 
+### Deploy locally
+After you succeed in running the NuGet Gallery, you can create a publish to deploy locally(such as your windows computer). 
+
+The steps are:
+1. Select the `NuGetGallery` project in Solution Explore of Visual Studio. 
+2. Right click the project, and then click `Publish` in the pop-up menu. Create a publish and make sure the Target is `Folder`
```
Comment (joelverhagen) [inline README.md:39]:
```suggestion
2. Right click the project, and then click `Publish` in the pop-up menu. Create a Publish configuration and make sure the Target is set to `Folder`.
```

File: README.md [hunk 3]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
...
+3. Copy the contents of the `Target Location` to any folder you want. Assume the folder is `C:\GreenSoftware\NuGetGallery`.
```
Comment (joelverhagen) [inline README.md:40]:
```suggestion
3. Copy the contents of the `Target Location` to any folder you want. For the following example, assume the folder is `C:\ContosoSoftware\NuGetGallery`.
```

File: README.md [hunk 4]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
...
+4. Run the command below to run it.(The parameter `/path` of iisexpress only supports absolute path on Windows)
```
Comment (joelverhagen) [inline README.md:41]:
```suggestion
4. Execute the command below to start the web app (note that the parameter `/path` of iisexpress.exe only supports absolute paths on Windows).
```

File: README.md [hunk 5]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
...
+    ```cmd
+    "C:\Program Files\IIS Express\iisexpress.exe" /path:C:\GreenSoftware\NuGetGallery
```
Comment (joelverhagen) [inline README.md:43]:
```suggestion
    "C:\Program Files\IIS Express\iisexpress.exe" /path:C:\ContosoSoftware\NuGetGallery
```

File: README.md [hunk 6]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
...
+    ```
+
+Now you can access the local website with browsers. The url is `https://localhost`
```
Comment (joelverhagen) [inline README.md:46]:
```suggestion
Now you can access the local website with a web browser. The URL is `https://localhost`.
```

File: README.md [hunk 7]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
 Refer to [our documentation](./docs/) for information on how to develop the frontend, use AAD, and more.
 
 ## Deploy
+### Deploy to Azure
 
 You will find instructions on how to deploy the Gallery to Azure [here](https://github.com/NuGet/NuGetGallery/blob/master/docs/Deploying/README.md).
 
+### Deploy locally
+After you succeed in running the NuGet Gallery, you can create a Publish configuration to deploy locally (such as your local Windows computer).
```
Comment (joelverhagen) [inline README.md:35]:
```suggestion
After you succeed in running the NuGet Gallery, you can create a publish profile to deploy locally (such as your local Windows computer). 
```

File: README.md [hunk 8]
```diff
@@ -27,9 +27,26 @@ Now run the NuGet Gallery:
...
+### Deploy to Azure
 
 You will find instructions on how to deploy the Gallery to Azure [here](https://github.com/NuGet/NuGetGallery/blob/master/docs/Deploying/README.md).
 
+### Deploy locally
+After you succeed in running the NuGet Gallery, you can create a Publish configuration to deploy locally (such as your local Windows computer). 
+
+The steps are:
+1. Select the `NuGetGallery` project in Solution Explore of Visual Studio. 
+2. Right click the project, and then click `Publish` in the pop-up menu. Create a Publish configuration and make sure the Target is set to `Folder`.
```
Comment (joelverhagen) [inline README.md:39]:
```suggestion
2. Right click the project, and then click `Publish` in the pop-up menu. Create a publish profile and make sure the Target is set to `Folder`.
```

**Review by joelverhagen — APPROVED** [review]
Awesome! Thank you!

**Review by joelverhagen — APPROVED** [review]
I've made some small changes to the wording and changed the base branch to `dev`.

**Comment by joelverhagen** [comment]
Thank you so much for your contribution @xiaoxstz! It's been merged to our `dev` branch. It will be in `main` soon.

**Pull request merged**